src/Yoti/Entity/Profile.php

```diff
@@ -106,24 +112,53 @@ public function getStructuredPostalAddress()
         return $this->getProfileAttribute(self::ATTR_STRUCTURED_POSTAL_ADDRESS);
     }
 
+    public function getDocumentDetails()
+    {
+        return $this->getProfileAttribute(self::ATTR_DOCUMENT_DETAILS);
+    }
+
     /**
-     * @return null|Attribute
+     * Return all derived attributes from the DOB e.g 'Age Over', 'Age Under'
+     * As a list of AgeVerification
+     *
+     * @return array
+     * e.g [
+     *      'age_under:18' => new AgeVerification(...),
+     *      'age_over:50' => new AgeVerification(...),
+     *      ...
+     * ]
      */
-    public function getAgeCondition()
+    public function getAgeVerifications()
```

Comment:
I understand that this method will return a collection of AgeVerification objects.
1. If there are no AgeVerification objects, maybe it would be better to return an empty list rather than _null_
2. We will still need to implement get-attribute-by-name in this Profile (and in the ApplicationProfile) i.e. a new method something like:
```
getAttribute(name) { ... }
```
When we do that, sdk users must still be able to call _getAttribute('age_over:18') and get the original attribute back (not the AgeVerification object).  Is that going to work with the approach in this PR?

Reply:
1. We want to be consistent with the other getters, if the attribute isn't set in the Dashboard we return NULL.

2. getAttribute(name){} is already implemented in BaseProfile::class and both Profiles class are extending this.